ovirt_imageio/client/_ui.py

```diff
@@ -101,24 +109,41 @@ def close(self):
 
     def _draw(self, last=False):
         line = self._format_line()
+        if self._format and not last:
+            line += ','
```

Comment:
Not needed, the output should be:

```
{"progress": 0, "description": "creating transfer", "done": 0, "elapsed": 0.0}
{"progress": 0, "description": "downloading image", "done": 0, "elapsed": 2.69}
```

Every line is a json object. The reader must read line by line and parse each
line separately. There are several specs describing this format, for example:
- https://jsonlines.org/
- http://ndjson.org/